common/scala/src/main/scala/org/apache/openwhisk/http/PoolingRestClient.scala

```diff
@@ -48,6 +50,8 @@ class PoolingRestClient(
   timeout: Option[FiniteDuration] = None)(implicit system: ActorSystem) {
   require(protocol == "http" || protocol == "https", "Protocol must be one of { http, https }.")
 
+  private val logging = new AkkaLogging(system.log)
```

Comment:
Initially, I was thinking about it, but it will require changing all implementations that extend or use `PoolingRestClient` and I think this is a backward incompatible change.

Reply:
Got it.
[The log store SPI](https://github.com/apache/openwhisk/blob/master/common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/logging/LogStore.scala#L101) does not pass an implicit parameter for logging.